File: DEVOPS/devops-tp/devops-ng/src/app/models/product.ts
export class Product {
  private _id: string = "";
  private _name: string = "";
  private _price: number = 0;

  public get id(): string {
    return this._id;
  }
  
  public set id(v: string) {
    this._id = v;
  }

  public get name(): string {
    return this._name;
  }

  public set name(v: string) {
    this._name = v;
  }
  
  public get price(): number {
    return this._price;
  }

  public set price(v: number) {
    this._price = v;
  }
}

File: DEVOPS/devops-tp/devops-java/src/main/java/fr/formation/model/Product.java
package fr.formation.model;

import java.math.BigDecimal;

import org.hibernate.annotations.UuidGenerator;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

@Entity
public class Product {
    @Id
    @UuidGenerator
    private String id;

    private String name;
    private BigDecimal price;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public BigDecimal getPrice() {
        return price;
    }

    public void setPrice(BigDecimal price) {
        this.price = price;
    }
}
